notebooks/MAST/TESS/interm_tesscut_astroquery/interm_tesscut_astroquery.ipynb

```diff
@@ -0,0 +1,507 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<a id=\"title_ID\"></a>\n",
+    "# Beginner: Cutout of the TESS FFIs using Astrocut and Astroquery\n",
+    "\n",
+    "This notebook shows the user how to use the MAST programmatic interface to create a cutout of the small section of the TESS FFIs. For this example we will determine the RA and Dec for TIC 261105201. We then perform a query to determine which sectors contain this RA and Dec, peform a cutout of the FFI time series, open the resulting target pixel files, and plot the first image of each file with the WCS overlayed on the image. Finally we will create a light curve from the resulting image by creating a photometric aperture and summing the light in our pixels.  \n",
+    "\n",
+    "This tutorial shows the users how to do the following: use astroquery.catalogs to query the TIC, use astroquery Tesscut to determine the number of sectors that contain our target and download a FFI cutout.\n",
+    "\n",
+    "\n",
+    "### Table of Contents \n",
+    "  [Astroquery Search of the TIC](#catalog_ID) <br> \n",
+    "  [Tesscut to Perform FFI Cutout](#tesscut_ID) <br>\n",
+    "  [Create Light Curve](#lightcurve) <br>\n",
+    "  [Additional Resources](#resources_ID) <br> \n",
+    "  [About this Notebook](#about_ID) "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "***"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Import Statements\n",
+    "<a id=\"imports_ID\"></a>\n",
+    "\n",
+    "We start with a few import statements.\n",
+    "- *numpy* to handle array functions\n",
+    "- *astropy.io fits* for accessing FITS files\n",
+    "- *astropy.wcs WCS* to interpret the World Coordinate Systems\n",
+    "- *matplotlib.pyplot* for plotting the data\n",
+    "- *astroquery.mast* for the catalogs and for tesscut.\n",
+    "\n",
+    "For ease of use later on, we also set the root url for the Tesscut package here.\n",
```

Comment:
Will this be removed? Or does this line refer to something other than setting the TESSCUT_URL?